languages/Cobol.groovy

```diff
@@ -17,10 +17,11 @@ println("** Building files mapped to ${this.class.getName()}.groovy script")
 // verify required build properties
 buildUtils.assertBuildProperties(props.cobol_requiredBuildProperties)
 
+// create language datasets
 def langQualifier = "cobol"
 buildUtils.createLanguageDatasets(langQualifier)
 
-if (props.runzTests == "True") {
+if (props.runzTests) {
```

Comment:
It is enough ? Or should we have this ?
if (props.runzTests && props.runzTests .toBoolean())